{
  "id": 5600852,
  "key": "MM-T2703",
  "name": "Incorrect Login ID Attribute",
  "createdOn": "2020-06-01T17:46:19Z",
  "objective": null,
  "precondition": null,
  "estimatedTime": null,
  "labels": ["se-prod", "ldap"],
  "customFields": {
    "Playwright": null,
    "Authors": "",
    "Priority P1 to P4": "P4 - Low-Impact (Edge or unsuitable to repeat?)",
    "Update notes": "",
    "Rainforest": ["N/A"],
    "Team Ownership": ["Suite Users"],
    "Manual Test Environments": [],
    "Detox": null,
    "Cypress": "To Do",
    "MMCTL": null,
    "Last Updated": "",
    "Tags": [],
    "Location": "AD LDAP"
  },
  "componentName": null,
  "priorityName": "Low",
  "statusName": "Active",
  "folderName": "AD LDAP",
  "folderFullPath": "suite/ad-ldap",
  "steps": [
    {
      "inline": {
        "description": "Incorrect Login ID Attribute<br>————————————————————————————<br><strong>Test using an LDAP account that does not yet have an account on this Mattermost server</strong><ol><li>Change Login ID Attribute to be something different, such as `domainusername` and Save</li><li>Have an LDAP user try to create a new account using LDAP credentials, observe error message \"We couldn't find an account matching your login credentials.\"</li><li>Change Login ID Attribute back to what it was before, such as `uid` or `sAMAccountName` and Save</li><li>Try the LDAP user account creation again</li></ol>",
        "testData": null,
        "expectedResult": "<ul><li>After switching back to correct Login ID Attribute such as sAMAccountName, account creation is successful</li></ul>",
        "customFields": {
          "Fix Versions": [],
          "Related ticket(s)": "",
          "Important notes about this step": ""
        }
      },
      "testCase": null
    }
  ],
  "caseHashed": "b2d389093a053874524f98db1587042657ddf8831f8ae65c299ad9b4a0516ae9e1a17a8b1ee96b3437bf69a53c8050b2",
  "stepsHashed": "7d278233253abe81b7286f43f0b9d23bd1de698746b06929c14117f8b6a500c1ae829790263ba8a16ad101ada8766faa"
}
